bangazonapi/models/product.py

```diff
@@ -62,7 +62,10 @@ def average_rating(self):
         for rating in ratings:
             total_rating += rating.rating
 
-        avg = total_rating / len(ratings)
+        try:
+            avg = total_rating / len(ratings)
+        except:
```

Comment:
Could you please specify the error you're handling here?

Reply:
Pushed new commit to address this.